handlers/commands/getItems.js

```diff
@@ -0,0 +1,51 @@
+const parse = require('urlencoded-body-parser')
+const fetch = require('node-fetch')
+
+const { send } = require('micro')
+
+module.exports = async (req, res) => {
+  const { team_domain, token, user_name } = await parse(req)
+  if (token !== process.env.SLACK_VERIFICATION_TOKEN) {
+    send(res, 401, 'Token did not equal verification token')
+    return
+  }
+
+  const email = `${user_name}@${team_domain}.com`
+  const url = `https://pantry-list-api.herokuapp.com/items?user=${email}`
+
+  const response = await fetch(url, {
+    headers: {
+      authorization: `Basic ${process.env.SLACK_TOKEN}`,
+      'Content-Type': 'application/json'
+    }
+  })
+
+  if (response.ok) {
+    const json = await response.json()
+    console.log(json.length)
+    if (json.length > 0) {
+      const items = json.map(i => {
+        let item = {
+          color: '#7c4be0',
+          title: `${i.item_name}`
+        }
+        if (i.item_details && i.item_details.description !== '') {
+          item.text = ` \n${i.item_details.description}`
+        }
+        return item
```

Comment:
Is there any way we can refactor this so that there aren't 3 nested `if` statements? Maybe 
```
if (!response.ok) { return send(res, ...) }
``` 
or something?

Reply:
Absolutely. I'll refactor this.